library/Rules/Tld.php

```diff
@@ -121,6 +121,12 @@ class Tld extends AbstractRule
 
     public function validate($input)
     {
-        return in_array(strtolower($input), $this->tldList);
+        // prefere LayerShifter\TLDDatabase if available
+        if (class_exists('\LayerShifter\TLDDatabase\Store')) {
+            $tldDb = new \LayerShifter\TLDDatabase\Store();
```

Comment:
Please import this class

Reply:
I've tried to import, but couldn't get it to work if TLDDatabase is not available. Maybe I did something wrong, because I'm beginner to OOP.